photutils/__init__.py

```diff
@@ -12,22 +12,13 @@
 from ._astropy_init import *  # noqa
 # ----------------------------------------------------------------------------
 
-# Enforce Python version check during package import.
-# This is the same check as the one at the top of setup.py
-import sys
+from pkg_resources import get_distribution, DistributionNotFound
```

Comment:
define `__all__`, then these won't be picked up by the API docs build (I think at least). having `__all__` is super nice anyway, so no other explicitly required thing will end up in the namespace either.